.github/workflows/open_data_dataloading.yml

```diff
@@ -0,0 +1,41 @@
+name: Open Data Refresh
+on:
+  workflow_dispatch:
+
+
+jobs:
+  dataload:
+    runs-on: ubuntu-20.04
+    env:
+      AWS_S3_ENDPOINT: ${{ secrets.DO_S3_ENDPOINT }}
+      AWS_ACCESS_KEY_ID: ${{ secrets.DO_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.DO_SECRET_ACCESS_KEY }}
+      AWS_S3_BUCKET: edm-recipes
+    strategy:
+      matrix:
+        dataset:
+          - dcp_cdboundaries_wi
+          - dcp_cb2010_wi
+          - dcp_ct2010_wi
+          - dcp_cb2020_wi
+          - dcp_ct2020_wi
+          - dcp_school_districts
+          - dcp_firecompanies
+          - dcp_councildistricts_wi
+          - dcp_healthareas
+          - dcp_healthcenters
+          - dcp_zoningmapindex
+          - dpr_greenthumb
+          - dsny_frequencies
+          - lpc_historic_districts
+          - lpc_landmarks
+          - dof_dtm
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: NYCPlanning/action-library-archive@v1.1
+        with:
+          name: ${{ matrix.dataset }}
+          s3: true
+          latest: true
+          output_format: pgdump
```

Comment:
My initial thought is that this is a nice improvement and would make our lives easier. My one concern is that the defaulting to pgdump could be an issue as these datasets might be read in different ways across our data products. Taking a look at` dcp_firecompanies` DO folder saw that there are multiple formats we output this particular dataset to
<img width="1041" alt="Screen Shot 2022-10-03 at 11 03 16 AM" src="https://user-images.githubusercontent.com/41444592/193611314-8904afc5-3034-4fd9-a2e5-6b5f3de422f3.png">

Reply:
Right that's true. Pluto only needs the .sql file, I think the most storage efficient approach is that if another dataset needs another version that other dataset runs the data library archive to get the file format it needs. But maybe we just just run csv, pgdump and shp to make it easier to run another pipeline

Reply:
Yeah for sure. curious to hear what @td928 and @AmandaDoyle think